src/Client/Resources/keen-io-3_0.php

```diff
@@ -1247,6 +1150,146 @@
+            'parameters'  => array(
+                'projectId'        => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'key'      => array(
+                    'location'    => 'uri',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+                'masterKey' => array(
+                    'location'    => 'header',
+                    'description' => 'The Master API Key.',
+                    'sentAs'      => 'Authorization',
+                    'pattern'     => '/^([[:alnum:]])+$/',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+            ),
+            'additionalParameters' => array(
+                'location' => 'json'
+            ),
+        ),
+        'revokeAccessKey' => array(
+            'uri'         => 'projects/{projectId}/keys/{key}/revoke',
+            'description' => 'Revokes a project access key.',
+            'httpMethod'  => 'POST',
+            'parameters'  => array(
+                'projectId'        => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'key'      => array(
+                    'location'    => 'uri',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+                'masterKey' => array(
+                    'location'    => 'header',
+                    'description' => 'The Master API Key.',
+                    'sentAs'      => 'Authorization',
+                    'pattern'     => '/^([[:alnum:]])+$/',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+            ),
+        ),
+        'unRevokeAccessKey' => array(
+            'uri'         => 'projects/{projectId}/keys/{key}/unrevoke',
+            'description' => 'Unrevokes a project access key.',
+            'httpMethod'  => 'POST',
+            'parameters'  => array(
+                'projectId'        => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'key'      => array(
+                    'location'    => 'uri',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+                'masterKey' => array(
+                    'location'    => 'header',
+                    'description' => 'The Master API Key.',
+                    'sentAs'      => 'Authorization',
+                    'pattern'     => '/^([[:alnum:]])+$/',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+            ),
+        ),
     ),
     'models' => array(
         'getResponse' => array(
```

Comment:
I think the key `addtionalProperties` has a syntax error here, should I fix it or it has to be like that?